documentation/decisions/adr/0003-api-language.md

```diff
@@ -0,0 +1,95 @@
+
+In addition to the Nava and HHS teams ability to develop and maintain an API in the chosen language, it is also important that this language lends itself to open source engagement.
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+- Nava and HHS familiarity with language and common libraries
+- Performance of language and associated frameworks at scale
+- Support of Open Source community
+- Cost of necessary licenses, support, etc.
+
+## Options Considered
+
+- Python
+- Javascript and Node
+- Java
+
+
+## Decision Outcome <!-- REQUIRED -->
+Chosen option: Python, because it is free and open source and designed to be easy for anyone to learn quickly and contribute. Additionally, while all the languages can support the technical needs, Python in particular is optimized to handle the large and complex grants.gov dataset, and makes other parts of the project simpler, including ETL and data analysis for the analytics endpoints. 
+
+### Positive Consequences <!-- OPTIONAL -->
+- We're hopeful that this choice will lead to open source contribution and adoption
+- We can make use of python libraries designed for complex data queries to quickly deliver features and analytics
+
+### Negative Consequences <!-- OPTIONAL -->
+- Will require some context switching when going from work on the API to the front-end
+
+
+## Pros and Cons of the Options <!-- OPTIONAL -->
+
+
+### Python
+
+Python as a scripting language is fast and productive. Web-based applications can be created quickly and the code is highly readable. Python syntax is easy to understand, well-defined and extensible. APIs built over Python are highly scalable and have an unmatched speed.
+
+Because of it's popularity in the open source community, Python makes it easy for open source contributors to support our work. 
+
+- **Pros**
+  - Nava and HHS experience with Python and APIs built with Python
+  - Number of available libraries and frameworks
+  - Robust data science and algorithmic libraries
+  - Free and open source to use
+  - Ideal for data computation and complex database queries
+- **Cons**
+  - Single threaded locking of GIL can make computationally expensive opersation slower than other options
+
+
+### Javascript and Node
+
+Node JS is used to build fast, highly scalable network applications based on an event-driven non-blocking input/output model, single-threaded asynchronous programming. It also suits well for microservice-based products. Such Node.js utility is accomplished by such characteristics as support of a non-blocking input-output and a significant number of connections.
+
+Because of it's popularity in the open source community, Javascript and Node makes it easy for open source contributors to support our work.
```

Comment:
nit: small grammar fix

```suggestion
Because of its popularity in the open source community, Javascript and Node makes it easy for open source contributors to support our work.
```